src/control/cmd/dmg/main.go

```diff
@@ -121,6 +122,19 @@ func exitWithError(log logging.Logger, err error) {
 	os.Exit(1)
 }
 
+func writeManPage(wr io.Writer) {
+	var opts cliOptions
+	p := flags.NewParser(&opts, flags.Default)
+	p.ShortDescription = "Administrative tool for managing DAOS clusters"
+	p.Usage = "dmg [OPTIONS] [COMMAND][SUBCOMMAND]"
```

Comment:
Seems like there should be a space between `[COMMAND] [SUBCOMMAND]`, no?

Reply:
I look at some other tools and they don't seem to do that but I'm not tied to it either way. I'll update it to have the space.